### PR TITLE
Add standard dashboard

### PR DESCRIPTION
**Pull request by davidrapson**
Thanks to @mattandrews for making this so easy. The changes here as mostly just cleanup and a few tweaks but this pretty much worked out of the box.

### CODE REVIEW

File: controllers/tools/applications.js [hunk 1]
```diff
@@ -202,14 +195,22 @@ function addCountry(row) {
...
 function getFeedbackDescriptionByAppId(appId) {
     let description;
-    switch (appId) {
-        case 'awards-for-all':
-            description = 'National Lottery Awards for All';
-            break;
-        default:
-            break;
+    if (appId === 'awards-for-all') {
+        description = 'National Lottery Awards for All';
```
Comment (mattandrews) [inline controllers/tools/applications.js]: Is this just the same mapping as `getApplicationTitle()` above?

Comment (davidrapson) [reply controllers/tools/applications.js]: It is, but accidentally I think, so made them separate functions.

Comment (davidrapson) [reply controllers/tools/applications.js]: Oh, on second thoughts it looks like the feedback description _is_ the form title. I can probably merge these then

**Comment by davidrapson** [comment]
@mattandrews Mind giving this another review. Looked at this a bit closer and there was a bug in the query that meant it wasn't correctly separating applications out and did some work to normalise the countries to use the actual `formBuilder` for pending forms. Applications with multiple locations are grouped by the same value we use in the summary (i.e. `uk-wide`)

**Comment by davidrapson** [comment]
[Create Your funding proposal dashboard](https://trello.com/c/cG6fQSlE/912-create-your-funding-proposal-dashboard)

File: db/models/applications.js [hunk 2]
```diff
@@ -77,17 +77,12 @@ class PendingApplication extends Model {
         });
     }
 
-    static findAllByForm(formId, dateRange = {}) {
-        let whereClause = {
-            formId: { [Op.eq]: formId }
-        };
-        if (dateRange.start && dateRange.end) {
-            whereClause = {
```
Comment (mattandrews) [inline db/models/applications.js]: Not sure how I missed this 🙈

**Pull request merged**